scripts/collect-status.js

```diff
@@ -3,8 +3,6 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-#! /usr/bin/env node
```

Comment:
Hmm, is it faulty because of the space between #! and /usr/bin/env? If so, can we just remove the space instead of removing the entire line? I like to just call it with ./scripts/collect-status.js and it seems to work for me.

Reply:
<img width="790" alt="Screenshot 2023-04-11 at 11 50 15" src="https://user-images.githubusercontent.com/1337046/231124240-961c1dd9-c23f-4922-b573-751602a49aef.png">

I think the shebang and the license not would battle for the position at the start of the file.